src/components/Modal/Confirm.vue

```diff
@@ -17,17 +18,20 @@ const props = defineProps({
 const emit = defineEmits(['reload', 'close']);
 
 const { send } = useClient();
+const { web3 } = useWeb3();
+const format = getChoiceString;
 
 const loading = ref(false);
+
 const symbols = computed(() =>
   props.strategies.map(strategy => strategy.params.symbol)
 );
-
-const format = getChoiceString;
+const walletConnectType = computed(() => web3.value.walletConnectType);
 
 async function handleSubmit() {
   loading.value = true;
-  await send(props.space.id, 'vote', {
+  const isSafe = walletConnectType.value === 'Gnosis Safe Multisig';
+  await send(props.space.id, isSafe ? 'vote-safe' : 'vote', {
```

Comment:
I've added a function "client.sign" in the client that does exactly like "client.broadcast" but without sending the tx. Can you try to use this function instead?

Reply:
Ok, @ChaituVR can you release a new version for this so I can use it?